### DISCUSSION TITLE
bookmarks do not track source edits [AutoHotkey]

### DISCUSSION THREAD

**Discussion by raffriff42** [alefragnani/vscode-bookmarks#747] [Ideas]
Steps to reproduce:
1) make a bookmark.
2) add (or remove) a line before the bookmark.
The bookmark does not move along with the code. 
This does not happen with breakpoints.
Am I missing something? 
Your extension is wonderful otherwise.

**Comment by raffriff42** [comment]
…and now it’s working. 😐

**Comment by alefragnani** [comment]
Hi @raffriff42 ,

The extension’s purpose is to update the bookmark position based on your edits. It moves the bookmark up if you delete lines above and moves the bookmark down if you add lines above. But, it is not bulletproof, and still presents a few issues in some scenarios, as you may see open issues about it. 

I’m planning to revisit this on my vacation, and expect to improve this

Hope this helps
